src/tests/ftest/daos_test/suite.py

```diff
@@ -57,6 +57,12 @@ def test_daos_pool(self):
         :avocado: tags=daos_test,daos_core_test,test_daos_pool
         """
         self.run_subtest()
+        try:
+            with open("/tmp/suite_dmg.log", 'r') as dmg_log:
+                log_data = dmg_log.readlines()
+                self.add_test_data("suite_pool_dmg.log", log_data)
+        except IOError as error:
+            print("unable to move dmg log:", error)
```

Comment:
Ideally we should pass `os.join(self.outputdir, f"{self.subtest_name}_dmg.log")` to the daos_test command to pass to the dmg `log-file` argument, so that we have something in the file if the test times out before it can move the dmg log file in /tmp, but if not this should be replaced with a simple mv:
```
os.rename("/tmp/suite_dmg.log", os.join(self.outputdir, f"{self.subtest_name}_dmg.log"))
```

Reply:
Can do. I'm going to need to create a separate PR to get that change into master, but I can update it for 2.2 here, since this is already an unclean cherry-pick.

Reply:
Whilst @phender is correct the use of f-stings is not valid in python2 so will likely mean that Test-tag for this test no longer works.

Reply:
I'm getting an error with this: `OSError: [Errno 18] Invalid cross-device link: '/tmp/suite_dmg.log' -> '/var/tmp/ftest/avocado/job-results/job-2022-08-16T08.52-5f13010/test-results/03-._daos_test_suite.py_DaosCoreTest.test_daos_pool_run-agent_config-transport_config-daos_tests-args-daos_test-num_clients-pools_created-scalable_endpoint-stopped_ranks-test_name-dmg-hosts-pool-server_config-servers-0-1-timeouts-abbc/data/DAOS_Pool_dmg.log'`

This error appears to be a result of using `os.rename` to copy a file from one filesystem to another. Apparently the workaround is to copy and delete. So I'll do that and fix the f-string.

Reply:
There's probably something in shlib to rename across filesystems.